### PR TITLE
nc-news-3 

### PR DESCRIPTION
**Pull request by taylorp2**
endpoints.json /api/

### CODE REVIEW

File: __tests__/server.test.js [hunk 1]
```diff
@@ -3,37 +3,49 @@ const app = require("../app");
...
+  test('responds with object (endpoint.json)', () => {
+    const endpointsJSON = require('../endpoints.json');
+    return request(app)
+    .get("/api")
+      .expect(200)
+      .then((response)=>{
+        expect(endpointsJSON).toEqual(response.body)
+      })
+  });
+});
```
Comment (leekli) [inline __tests__/server.test.js:51]: - Good to see you testing for the endpoints
- It's good that you're doing the request to `/api` and then checking the response against the `endpoints.json` file
- Requiring it in is file, you could also use `fs.readFile` but this is fine too. No changes needed here.
- I would recommend you add the expected status code in the test description too, and maybe the endpoints.json doesn't need to be explicitly stated in the description

File: controllers/controller.js [hunk 2]
```diff
@@ -7,3 +8,8 @@ exports.getTopics = (request, response) => {
   });
 };
 
+exports.getEndpoints=(req,response)=>{
+    const endpointsString = fs.readFileSync('endpoints.json', "utf-8");
+    const endpointsJSON = JSON.parse(endpointsString);
+    response.status(200).send(endpointsJSON);
+}
```
Comment (leekli) [inline controllers/controller.js:15]: Good to see you've set up this controller.

The function name `getEndpoints` is a good function name for this controller.

**Review by leekli — APPROVED** [review]
Hi Pete, well done on this.

I've left feedback throughout.

You are free to merge this branch.

File: __tests__/server.test.js (hunk 1 above)
Comment (taylorp2) [reply __tests__/server.test.js:51]: thanks for the feedback i renamed the describe and test

**Pull request merged**